app/classification.js

```diff
@@ -79,8 +84,30 @@ module.exports.getSemilarType = function(word) {
     .then((results) => {
       return results.reduce((finalType, result) => {
         return (result.similarity > finalType.similarity) ? result : finalType;
-      }, {type: null, similarity: 0})
+      }, {type: null, similarity: config.semilarAlgorithmThreshold})
     }).catch(() => {
       return {type: null, similarity: 0};
     });
 };
+
+// return string without nouns, adjectives, or adverbs
+exports.filterMeaningfulVerb = function (relation) {
+  let words = relation.split(' ');
+  let customWordsToExclude = ['was', 'were', 'to', 'be', 'am', 'is', 'are', 'also', 'then', 'had', 'has', 'have'];
```

Comment:
I came up with it myself. If you find an official list, feel free to include it!